psydac/api/basic.py

```diff
@@ -30,10 +30,12 @@ def __init__(self, expr, **kwargs):
                        or PSYDAC_BACKENDS['python']
 
         namespace = kwargs.pop('namespace', globals())
-        backend   = kwargs.pop('backend', default_backend)
+        backend   = kwargs.pop('backend', None)
         folder    = kwargs.pop('folder', None)
         comm      = kwargs.pop('comm', None)
         root      = kwargs.pop('root', None)
+
+        if backend is None:backend = default_backend
```

Comment:
Good catch! Please make it a one-liner:
```suggestion
        backend   = kwargs.pop('backend', None) or default_backend
        folder    = kwargs.pop('folder', None)
        comm      = kwargs.pop('comm', None)
        root      = kwargs.pop('root', None)
```

Reply:
ah good idea

Reply:
done